packages/core/test/specHelper.ts

```diff
@@ -167,7 +165,15 @@ class StubXhr {
     this.response = response
     this.status = status
     this.readyState = XMLHttpRequest.DONE
-    this.onreadystatechange()
+
+    if (this.isOnreadystatechangeAttributeCallFirst) {
+      this.onreadystatechange()
+      this.dispatchEvent('readystatechange')
```

Comment:
do we listen for this event somewhere?
if not, what about not handling it?

Reply:
Right, it's no logger needed